package.json

```diff
@@ -24,24 +24,25 @@
     "clean": "rm -rf ./src/generated ./dist ./build",
     "pretest": "npm run clean && ./scripts/regenerate_protos.sh && npm run lint && tsc --noEmit",
     "lint": "eslint . --ext .ts --fix",
+    "lintNoFix": "eslint . --ext .ts",
     "test": "nyc mocha",
     "prepublishOnly": "npm run test && npm run build"
   },
   "devDependencies": {
     "@types/chai": "4.2.11",
     "@types/google-protobuf": "^3.7.2",
     "@types/mocha": "7.0.2",
-    "@typescript-eslint/eslint-plugin": "^2.16.0",
-    "@typescript-eslint/parser": "^2.16.0",
+    "@typescript-eslint/eslint-plugin": "^3.0.2",
+    "@typescript-eslint/parser": "^3.0.2",
+    "@xpring-eng/eslint-config-base": "^0.3.2",
     "chai": "4.2.0",
     "copyfiles": "^2.2.0",
-    "eslint": "^6.8.0",
-    "eslint-config-airbnb-base": "^14.0.0",
-    "eslint-config-airbnb-typescript": "^7.0.0",
-    "eslint-config-prettier": "^6.9.0",
+    "eslint": "^7.0.0",
     "eslint-plugin-import": "^2.20.0",
-    "eslint-plugin-mocha": "^6.2.2",
+    "eslint-plugin-jsdoc": "^25.4.2",
```

Comment:
`@xpring-eng/eslint-config-base` doesn't actually have the JSDoc rules enabled yet, but it will soon.